cloud-gateway/src/main/java/com/registry/cloudgateway/controller/CloudGatewayController.java

```diff
@@ -0,0 +1,49 @@
+package com.registry.cloudgateway.controller;
+
+import com.registry.cloudgateway.dto.ClientDTO;
+import com.registry.cloudgateway.dto.InvoiceDTO;
+import com.registry.cloudgateway.service.CloudGatewayService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cloud.netflix.eureka.EnableEurekaClient;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@EnableEurekaClient
+@RestController
+public class CloudGatewayController {
+
+    @Autowired
+    private CloudGatewayService cloudGatewayService;
+
+    @GetMapping("/client/{id}")
+    public ClientDTO getClientById(@PathVariable("id") String clientNumber) {
+        return cloudGatewayService.findOneByClientNumber(clientNumber);
+    }
+
+    @GetMapping("/all-clients")
```

Comment:
Avoid using anything other than the collection name. I think this uri should be just clients.